backend/lib/spark-api-submit-deals.js

```diff
@@ -112,7 +116,7 @@ const markDealsAsSubmitted = async (pgPool, eligibleDeals) => {
  *
  * @param {string} sparkApiBaseURL
  * @param {string} sparkApiToken
- * @param {Array} deals
+ * @param {Array<Static<typeof SubmittableDeal>>} deals
```

Comment:
```suggestion
 * @param {SubmittableDeal[]} deals
```

Same [comment ](https://github.com/CheckerNetwork/spark-deal-observer/pull/95/files#r1952235257) is applicable here.